MREUnityRuntime/MREUnityRuntimeLib/App/MixedRealityExtensionApp.cs

```diff
@@ -577,8 +577,12 @@ private void OnCreateFromPrefab(CreateFromPrefab payload, Action onCompleteCallb
         {
             try
             {
-                var createdActors = _assetLoader.CreateFromPrefab(payload.PrefabId, payload.Actor?.ParentId);
-                ProcessCreatedActors(payload, createdActors, onCompleteCallback);
+                MREAPI.AppsAPI.AssetCache.OnCached(payload.PrefabId, _ =>
+                {
+                    if (this == null || _appState != AppState.Running) return;
```

Comment:
>appState != AppState.Running [](start = 41, length = 28)

The CodeFlow comment thread for this line got undocked and is at the file of this file now.